packages/mobx-react/package.json

```diff
@@ -51,8 +51,9 @@
         }
     },
     "devDependencies": {
-        "mobx": "^6.7.0",
-        "mobx-react-lite": "^3.4.0"
+        "mobx": "^6.8.0",
+        "mobx-react-lite": "^3.4.3",
```

Comment:
Should become 6.9?.0 and 4.0.0 at the time of release. Again not sure how to handle that.